File: core/common/src/main/java/com/blood/common/util/ToastUtil.java
package com.blood.common.util;

import android.content.Context;
import android.widget.Toast;

public class ToastUtil {

    public static void toast(Context context, String msg) {
        Toast toast = Toast.makeText(context, "", Toast.LENGTH_SHORT);
        toast.setText(msg);
        toast.show();
    }

}

File: high/filter/src/main/java/com/blood/filter/util/ToastUtil.kt
package com.blood.filter.util

import android.widget.Toast
import com.blood.filter.App

object ToastUtil {

    fun toast(msg: String?) {
        App.instance.runUITask {
            val toast: Toast = Toast.makeText(App.instance, "", Toast.LENGTH_SHORT)
            toast.setText(msg)
            toast.show()
        }
    }

}